projects/wave-core-new/src/lib/users/modal-login/modal-login.component.html

```diff
@@ -0,0 +1,11 @@
+<form [formGroup]="loginForm" (ngSubmit)="login()">
```

Comment:
Shouldn't we add a header here?

For other things like the lineage popup (https://github.com/geo-engine/geoengine-ui/blob/main/projects/wave-core-new/src/lib/provenance/lineage-graph/lineage-graph.component.html) and the rename popup (https://github.com/geo-engine/geoengine-ui/blob/main/projects/wave-core-new/src/lib/layers/rename-layer/rename-layer.component.html), there is always one.